controllers/operandrequest/reconcile_operand.go

```diff
@@ -209,27 +213,32 @@ func (r *Reconciler) reconcileCRwithConfig(ctx context.Context, service *operato
 			k8sRes.SetName(res.Name)
 			k8sRes.SetNamespace(k8sResNs)
 
-			err := r.Client.Get(ctx, types.NamespacedName{
-				Name:      res.Name,
-				Namespace: k8sResNs,
-			}, &k8sRes)
+			verbs := []string{"create", "delete", "get", "update"}
```

Comment:
@Daniel-Fan Do we need to check the permission for list and watch since we are using `client.get`?

Reply:
@horis233 Sorry for the late reply.....I did not notice the comment.

I created a ClusterRole and bind it to ODLM SA
```
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: odlm_configmap
rules:
  - verbs:
      - delete
      - update
      - create
      - get
    apiGroups:
      - ''
    resources:
      - configmaps
```
ODLM is able to create ConfigMap in other namespaces after the permission check.

Those verbs are enough to perform `Get`, `Update` and `Delete` from `Go-Client`.